Sources/App/Websocket/RoomSession.swift

```diff
@@ -0,0 +1,12 @@
+import Vapor
+
+struct RoomSession: Content, Hashable {
+  let id: String
+    //TODO add card deck
+}
+
+extension RoomSession: Parameter {
+  static func resolveParameter(_ parameter: String, on container: Container) throws -> RoomSession {
```

Comment:
The reason we don't need this line in other extension of Parameter is because they are SQLModels?

thought we had: `extension SomeClass: Parameter {}`

Reply:
make empty extension